rhoni-team/AppCI
# Calculator component

closes #37 

⚠️  This branch depends on the #42 issue, so I think we first have to merge that one.

The calculator for isolation end date has been created. It has the 'start date' and the 'isolation time' with its 'unit time' as inputs. Then it calculates the 'isolation end date' as output.

> How to test?

Install npm dependencies:


```bash
#@frontend/$
cd frontend
npm install
```

Please try some values and check if it works properly :godmode:
